src/polar/Pie.tsx

```diff
@@ -202,35 +205,35 @@ type PieComposedData = PieCoordinate & {
   sectors: PieSectorDataItem[];
   data: RealPieData[];
 };
-type PiePayloadInputProps = {
-  sectors: PieSectorDataItem[];
-  legendType?: LegendType;
-};
-
-const computeLegendPayloadFromPieData = ({ sectors, legendType }: PiePayloadInputProps): Array<LegendPayload> => {
-  if (sectors == null) {
-    return [];
-  }
-  return sectors.map(
-    (entry: PieSectorDataItem): LegendPayload => ({
-      type: legendType,
-      value: entry.name,
-      color: entry.fill,
-      payload: entry,
-    }),
-  );
-};
-
-function SetPiePayloadLegend(props: PiePayloadInputProps): null {
-  useLegendPayloadDispatch(computeLegendPayloadFromPieData, props);
-  return null;
-}
+// type PiePayloadInputProps = {
+//   sectors: Readonly<PieSectorDataItem[]>;
+//   legendType?: LegendType;
+// };
+
+// const computeLegendPayloadFromPieData = ({ sectors, legendType }: PiePayloadInputProps): Array<LegendPayload> => {
+//   if (sectors == null) {
+//     return [];
+//   }
+//   return sectors.map(
+//     (entry: PieSectorDataItem): LegendPayload => ({
+//       type: legendType,
+//       value: entry.name,
```

Comment:
How hard would it be to recreate this using the raw `displayedData` instead of sectors? If I read it correctly then it might be as simple as

1.
```
value: getValueByDataKey(entry, dataKey, 0),
name: getValueByDataKey(entry, nameKey, i),
color: entry.fill,
payload: entry // this part will be breaking but w/e it was never documented anyway
```

Reply:
Yeah I was trying to get away without the breaking change for whatever reason. 

There's more info in entry in Pie than some other cases - for example "percentage" is a computed property that exists on sector that exists only to give more data to custom tooltip and legend

Reply:
If I did this though I wouldn't be able to get the `Cell` `fill` for use in `Legend`, another breaking change which is a pretty big use-case of cell

Reply:
I think scatter is broken in this way too